app/lib/actions/pulp3/orchestration/repository/import_upload.rb

```diff
@@ -48,7 +48,14 @@ def run
             if input[:content_unit_href]
               output[:content_unit_href] = input[:content_unit_href]
             elsif input[:artifact_output]
-              output[:content_unit_href] = input[:artifact_output][:content_unit_href] || input[:artifact_output][:pulp_tasks].last[:created_resources].first
+              content_unit_href = input[:artifact_output][:content_unit_href]
+              output[:content_unit_href] = if content_unit_href
```

Comment:
why do we additionally have this? I would have expected to have something like:
`
if repo.deb?
                                             input[:artifact_output][:content_unit_href] || input[:artifact_output][:pulp_tasks].last[:created_resources].find { |href| href.include?("/deb/packages") }
                                           else
                                             input[:artifact_output][:content_unit_href] || input[:artifact_output][:pulp_tasks].last[:created_resources].first
                                           end
`

Reply:
Oh, just saw we cannot use `repo.deb?`, because repo is not defined in that method :grimacing: 

I guess we have to assume that only deb-repos have `deb/packages`-content :crossed_fingers: